packages/teams-js/src/internal/communication.ts

```diff
@@ -841,8 +858,12 @@ function createMessageRequest(
   func: string,
   args: any[] | undefined,
 ): MessageRequestWithRequiredProperties {
+  const messageId: MessageID = CommunicationPrivate.nextMessageId++;
```

Comment:
Sort of, this is mostly so that if we have a Host that's using an older version of their codebase this change wouldn't impact them. Internally we are using UUIDs, but upon sending we still need to account for how the host will process these messages and have to add on the number as a messageID